### PR TITLE
Feature/debug frame

### PR DESCRIPTION
**Pull request by blooop**
Added a feature to return the values of the created debug lines to give the user the option of deleting them individually later

Also added type hints.

### CODE REVIEW

File: pyb_utils/frame.py [hunk 1]
```diff
@@ -54,25 +65,30 @@ def debug_frame(size, obj_uid, link_index):
     link_index : int
         Index of the link on the body to attach the frame to. Use `-1` for the
         base link.
+    Returns
+    -------
+    :
```
Comment (adamheins) [inline pyb_utils/frame.py:70]: nitpick: for return part of the docstring, the format is
```
Returns
-------
: <type>
    <description>
```
can you change that here and in the `debug_frame_world` docstring as well?

**Review by adamheins — CHANGES_REQUESTED** [review]
Looks good, with just two nitpicks with the docstrings. Once these are fixed we can merge.

Adding type hints is something I've had on the todo list for a while, so it's good that you're starting that off here.

File: pyb_utils/frame.py (hunk 1 above)
Comment (blooop) [reply pyb_utils/frame.py]: fixed

Comment (adamheins) [reply pyb_utils/frame.py]: can you also remove the colon in front of the description part? there should only be a colon before the type

Comment (blooop) [reply pyb_utils/frame.py]: sorry, removed now.

Comment (adamheins) [reply pyb_utils/frame.py]: thanks!

**Pull request merged**